webdriver/tests/bidi/script/get_realms/get_realms.py

```diff
@@ -0,0 +1,178 @@
+import pytest
+
+from webdriver.bidi.modules.script import ContextTarget
+
+from ... import any_string, recursive_compare
+
+PAGE_ABOUT_BLANK = "about:blank"
+
+
+@pytest.mark.asyncio
+async def test_payload_types(bidi_session):
+    result = await bidi_session.script.get_realms()
+
+    recursive_compare(
+        [
+            {
+                "context": any_string,
+                "origin": any_string,
+                "realm": any_string,
+                "type": any_string,
+            }
```

Comment:
We do know some of the fields:
```suggestion
            {
                "context": top_context["context"],
                "origin": any_string,
                "realm": any_string,
                "type": "window",
            }
```

Reply:
So here the idea from Henrik was to make this test very basic to not overlap with other tests